pipeline/terraform/2024-01-09/main.tf

```diff
@@ -2,22 +2,22 @@ module "pipeline" {
   source = "../modules/stack"
 
   reindexing_state = {
-    listen_to_reindexer      = false
-    scale_up_tasks           = false
-    scale_up_elastic_cluster = false
-    scale_up_id_minter_db    = false
-    scale_up_matcher_db      = false
+    listen_to_reindexer      = true
```

Comment:
Does all this need to be scaled back down now?

Reply:
Yes - will do that now